central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl.go

```diff
@@ -32,23 +32,23 @@ type datastoreImpl struct {
 
 // GetScanConfiguration retrieves the scan configuration specified by id
 func (ds *datastoreImpl) GetScanConfiguration(ctx context.Context, id string) (*storage.ComplianceOperatorScanConfigurationV2, bool, error) {
-	if ok, err := scanConfigurationsSAC.ReadAllowed(ctx); err != nil {
-		return nil, false, err
-	} else if !ok {
+	scanConfig, found, err := ds.storage.Get(ctx, id)
```

Comment:
Not sure why the order was flipped here?

Reply:
@theencee Because the way the SAC filter works, I felt it best to do it that way.

For example, we have a scan configuration that that applies a schedule to Cluster 1, Cluster 2, and Cluster 3.

The user requests the scan configuration but only has access to Cluster 1.

I'm not sure it makes sense to show the user the configuration at all at least short term.  At that point it becomes complicated on figuring out what to show them on the configuration object.  Especially since clusters is a child join table of the configuration, they all get returned with the serialized object.

Reply:
Oh I missed that scanConfig is needed to get cluster keys